examples/planning/closed_loop_test.ipynb

```diff
@@ -0,0 +1,347 @@
+    "\n",
+    "We refer to this process with the terms **forward-simulate** or **unroll**.\n",
+    "\n",
+    "![closed-loop](../../images/planning/closed-loop.svg)\n",
+    "\n",
+    "\n",
+    "## What can we use closed-loop evaluation for?\n",
+    "Closed-loop is crucial to assess the real performance of a model. **Ideally, we would test the model on the road in the real world**. However, this is clearly very expensive and scales poorly. Forward-simulation is an attempt to evaluate the system in a setting which is as close as possible to a real road test on the same route.\n",
+    "\n",
+    "Differently from the open-loop, the model is now in full control of the SDV and predictions are used to compute the future location of the SDV.\n",
+    "\n",
+    "## Is closed-loop evaluation enough?\n",
+    "Closed-loop evaluation is an important step forward towards evaluating how our policy would perform on the road. However, it still have some limitations.\n",
+    "\n",
+    "The critical one is the **non-reactivity of other traffic participants**. In fact, while the SDV is now controlled by our policy, other actors are still being replayed from the original log. In this setting, a chasing car will not slow down if our policy choses a different speed profile for the SDV, resulting in a rear collision that wouldn't obviously occur in the real world.\n",
+    "\n",
+    "For this reason, closed-loop evaluation is only effective for the first few seconds of forward simulation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from tempfile import gettempdir\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "from torch import nn, optim\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torch.utils.data.dataloader import default_collate\n",
+    "from tqdm import tqdm\n",
+    "\n",
+    "from l5kit.configs import load_config_data\n",
+    "from l5kit.data import LocalDataManager, ChunkedDataset, filter_agents_by_frames\n",
+    "from l5kit.dataset import EgoDataset\n",
+    "from l5kit.rasterization import build_rasterizer\n",
+    "from l5kit.geometry import transform_points, angular_distance, yaw_as_rotation33\n",
+    "from l5kit.visualization import TARGET_POINTS_COLOR, PREDICTED_POINTS_COLOR, draw_trajectory\n",
+    "from l5kit.planning.utils import detect_collision\n",
+    "from l5kit.kinematic import AckermanPerturbation\n",
+    "from l5kit.random import GaussianRandomGenerator\n",
+    "\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare data path and load cfg\n",
```

Comment:
Shouldn't this already be done by the user when above we point to 'has to set up data path properly ...'
(guess we can't skip the cell, but maybe shorten text / compress cells to "setup as mentioned above ..")

Reply:
this is more about linking it in the notebook